uint32 id
geometry_msgs/Pose pose
uint8 CREATION_TYPE_MANUAL=0
uint8 CREATION_TYPE_AUTO=1
uint8 creation_type
string name 